src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -164,6 +164,22 @@ def _row_to_doc(row, kbase_collection, load_version, features, tool_genome_map,
     return doc
 
 
+def _create_tool_upa_map(tool_identifiers, source_dirs, meta_filenames):
+    # Build a hash map between tool identidier and meta file path
+    res = dict()
+    for tool_identifier, source_dir, meta_filename in zip(tool_identifiers, source_dirs, meta_filenames):
+        upa_dict = {}
+        if not pd.isna(meta_filename):
```

Comment:
I would name it as `meta_df` if it's a dataframe. Otherwise, I would think `meta_filename` is a string or Path.

Reply:
I already reverted back. mete_filename is indeed a string or path